App.js

```diff
@@ -1,26 +1,45 @@
-import React from 'react'
-import styled from 'styled-components/native'
+import React, { useState, useEffect } from "react";
 
-const Container = styled.View`
+import { Firstpage } from "./components/Firstpage";
```

Comment:
You need to be consistent in your naming, the file is called `FirstPage.js` but everywhere else you call it `Firstpage`. PascalCase is the way to go 👍